all2vec/__init__.py

```diff
@@ -96,13 +107,26 @@ def __init__(self, nfactor):
         self._annoy_objects = {}
         self._entity_id_map = {}
 
+    def get_size(self):
+        """Return size of set."""
+        return len(self._annoy_objects)
+
     def create_entity_type(self, entity_type_id, entity_type,
                            ntrees, metric='angular'):
         """Create an entity type and populate its metadata."""
         self._annoy_objects[entity_type] = EntityType(
             self._nfactor, ntrees, metric, entity_type_id, entity_type)
         self._entity_id_map[entity_type_id] = entity_type
 
+    def get_entity_type(self, entity_type_id):
+        """Get entity-type for specified id."""
+        entity_type = self._entity_id_map[entity_type_id]
+        return self._annoy_objects[entity_type]
+
+    def get_nfactor(self):
```

Comment:
No need to change this, but in Python, you can use the `@property` decorator to mimic getters/setters in Java:

```
@property
def nfactor(self):
    return self._nfactor
```

You can then access `es.nfactor` without granting access to the private attribute.